### DISCUSSION TITLE
下载清晰度，B站在先观看1080，下载480  传了cookie

### DISCUSSION THREAD

**Discussion by ybsdegit** [yutto-dev/yutto#160] [Q&A]
yutto BV16h4y1E79m -d D:\Photograph\Bilibili -c "buvid3=ECE9CD02-8320-166D-A40E-59425.................................."

>INFO  * 0 [AVC ] [ 768x480 ] <480P 清晰 > #3

**Comment by SigureMo** [comment]
> yutto BV16h4y1E79m -d D:\Photograph\Bilibili -c "buvid3=ECE9CD02-8320-166D-A40E-59425.................................."

注意一下 cookie 的格式，仔细看下文档是怎么写的
